spec/factories/posts.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :post do
+    user
+    section
```

Comment:
I think we need to specify the `association` and have factories for those corresponding posts if we want it to be populated.